website/docs/docs/cloud/git/setup-azure.md

```diff
@@ -62,22 +62,18 @@ An Entra ID admin needs to provide your new app access to Azure DevOps:
 
 ## Add another redirect URI
 
-A Microsoft Entra ID admin needs to add another redirect URI to your Entra ID application. This redirect URI will be used to authenticate the service user for headless actions in deployment environments.
```

Comment:
On step number 6 above it says ` Add a redirect URI by selecting...` are we leaving that in place for service user or is it not needed now?

Reply:
I guess we should remove all of that.